app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -60,40 +64,30 @@ keywords:
-
-## Why train to teach in England?
+QTS will allow you to apply for teaching jobs in all schools in England. When you get a teaching job, depending on your immigration status, you may also need to [apply for a visa allowing you to work in the UK](/non-uk-teachers/visas-for-non-uk-teachers).   
 
-Teacher training in England is practical, hands-on and supportive. As an international applicant, you’ll have the opportunity to train at a world class institution where creativity and innovation is valued, gaining lots of classroom experience along the way.
+### Meet non-UK citizens who trained to teach in England
 
-When you have successfully completed your training, you will be awarded [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts), a professional qualification valued globally.
+$why-train-video$
 
-QTS will allow you to apply for teaching jobs in all schools in England.    
+[Read the transcript for the 'Why train to teach in England?' video](/non-uk-teachers/why-train-to-teach-in-england-video-transcript).
 
-[Read teachers' stories about the rewards of training to teach in England](/blog).
+## Useful info
 
-## Other ways to train
+When you plan your training, bear in mind that:
 
-You can also apply to [train to teach in Wales](https://educators.wales/home), [train to teach in Scotland](https://teachinscotland.scot/) or [train to teach in Northern Ireland](https://www.education-ni.gov.uk/articles/initial-teacher-education-courses-northern-ireland).
-
-If you don’t want to come to England to study you can apply to train for international qualified teacher status (iQTS). This meets the same high standards as [English qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts) and, if successfully completed, leads to the automatic award of QTS.
-
-For teachers who qualified outside the UK, as well as those with teaching experience, [other routes to qualified teacher status (QTS)](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk/routes-to-qualified-teacher-status-qts-for-teachers-and-those-with-teaching-experience-outside-the-uk) are also open.
+- international candidates can expect to pay an average of about £15,000 to train, plus living costs
+- depending on your immigration status and the subject you train to teach, you may get some financial support
+- training lasts one academic year full-time, or longer part-time
+- you can apply for paid teaching jobs as soon as you successfully complete your training and gain English qualified teacher status (QTS)
+- you'll earn at least £30,000 a year (or more in London) when you qualify as a teacher 
+- you may need to extend or switch your visa to live and work in the UK long-term
 
-## Check your qualifications and get help
+## 1. Check your qualifications
```

Comment:
Do these steps need to be done in this order?

Reply:
yes :)